app/assets/stylesheets/modules/breakpoints.scss

```diff
@@ -5,8 +5,13 @@
 		@content;
 	}
 }
+@mixin for-phone-and-tablet-only {
+	@media (max-width: 899px) {
+		@content;
+	}
+}
 @mixin for-tablet-only {
-	@media (min-width: 600px) and (max-width: 1200px) {
+	@media (min-width: 600px) and (max-width: 1199px) {
```

Comment:
this was broken so I fixed it - not even sure I ended up using it but it would have made things appear double so have done it anyway!